src/tye/Program.UndeployCommand.cs

```diff
@@ -24,24 +25,58 @@ public static Command CreateUndeployCommand()
                 StandardOptions.Verbosity,
                 StandardOptions.Tags,
 
+                new Option(new string[]{"-f", "--framework" })
```

Comment:
You can put this in the StandardOptions instead of duplicating it for all commands.

Reply:
I though about that but there's a small issue 
there's 2 Command that uses `-f` for `--force` 
do I simply remove `-f` for `--framework` ?

PS:
one of the 2 command has it declared / passed down but never use it
should i also remove it  ?

Reply:
Bleh, that unfortunate. @davidfowl @JunTaoLuo should we change the -f universally to be --framework and change --force to something else?

Reply:
I think it makes sense to use -f for --framework just to keep in line with the rest of dotnet commands (like dotnet run). How about leaving --force without a short option?

Reply:
would prefer `--force` rather than `-force` for consistency over all CLI in the `dotnet` echo system TBH

Reply:
done but it would require a review:
* I moved all `ForceOptions` to StandardOptions.CreateForce("description here")`
* There's no `-f` anymore on that `CreateForce), so it kinda break few commands which might not be ideal

The idea would to be able to check if there's already `Standard.Framework` and depending on it use `-f` for `--force`
Doing this would have a bad side effect if in some "month/year", for ... reason `--framework` appear, then the `-f` would change from `force` to `framework`